pom.xml

```diff
@@ -874,6 +891,11 @@ THE SOFTWARE.
                     <bannedImport>org.apache.commons.httpclient.**</bannedImport>
                   </bannedImports>
                 </restrictImports>
+                <bannedDependencies>
```

Comment:
please change the id as well 
`<id>ban-commons-httpclient-3x</id>`
to 
`<id>ban-commons-httpclient-3x-and-commons-lang</id>`
to have more consistency.

Reply:
changed to new execution rule.